src/main/java/org/symphonyoss/symphony/messageml/elements/Option.java

```diff
@@ -0,0 +1,50 @@
+package org.symphonyoss.symphony.messageml.elements;
+
+import org.symphonyoss.symphony.messageml.exceptions.InvalidInputException;
+import org.symphonyoss.symphony.messageml.markdown.nodes.OptionNode;
+
+import java.util.Collections;
+
+/**
+ * Class representing a Symphony Elements option
+ *
+ * @author lumoura
+ * @since 03/22/19
+ */
+public class Option extends Element {
+
+  public static final String MESSAGEML_TAG = "option";
+  public static final String VALUE_ATTR = "value";
+
+  public Option(Element parent) {
+    super(parent, MESSAGEML_TAG);
+  }
+
+  @Override
+  protected void buildAttribute(org.w3c.dom.Node item) throws InvalidInputException {
+    if (item.getNodeName().equals(VALUE_ATTR)) {
+      setAttribute(VALUE_ATTR, getStringAttribute(item));
+    } else {
+      throw new InvalidInputException("Attribute \"" + item.getNodeName()
+              + "\" is not allowed in \"" + getMessageMLTag() + "\"");
+    }
+  }
+
+  @Override
+  public org.commonmark.node.Node asMarkdown() {
+    return new OptionNode();
+  }
+
+
+  @Override
+  public void validate() throws InvalidInputException {
+    if (this.getParent().getClass() != Select.class) {
```

Comment:
You shouldn't be doing this here. We have a method on the Elements class which validates a Parent based on what you accepted. See https://github.com/symphonyoss/messageml-utils/blob/master/src/main/java/org/symphonyoss/symphony/messageml/elements/CardBody.java for more details on how to implement it.